docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,350 @@
+.. _Courses API Blocks Resource:
+
+########################################
+Courses API Blocks Resource
+########################################
+
+With the Courses API **Blocks** resource, you can complete the
+following tasks.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _Get a list of the course blocks in a course:
+
+*******************************************************
+Get a List of Course Blocks in a Specified Course
```

Comment:
Get a List of Course Blocks in a Specific Course

or

Get a List of Course Blocks in a Course

I think "specific" course sounds more natural than "specified." Maybe we can skip the word entirely?